src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -271,6 +271,39 @@
      * or enum declaration (interfaces cannot have static initializers).  The
      * first and only child is a statement list.
      *
+     * <p>For example:</p>
+     *
+     * <pre>
+     * static int num;
```

Comment:
I think it is ok to avoid this sentence in example of javadoc.

Reply:
Done

Reply:
I still see this line in code changes, please remove

Reply:
My bad, I thought you mentioned about "for example". 
Done, `static int num` is removed now.